cellprofiler/gui/figure.py

```diff
@@ -1493,6 +1493,15 @@ def subplot_imshow_color(self, x, y, image, title=None, normalize=False, rgb_mas
         if rgb_mask is None:
             rgb_mask = [1, 1, 1]
 
+        # Truncate multichannel data that is not RGB(A) (5+ channel data) and display it as RGB.
```

Comment:
So just to clarify - CellProfiler will accept >3 channel 2D images and process, it simply won't display all the channels as RGB?

Reply:
Correct. The first 3 channels will be displayed as RGB. Does this seem reasonable?

Reply:
I think it does, especially with the warning message.

Reply:
Cool. Let's see if @bethac07 agrees.